contracts/ICO_controller.sol

```diff
@@ -328,11 +271,13 @@ contract ICO_controller is Ownable, TransferableInterface {
         return false;
     }
 
-    function sendPullTokensTo(address _to, uint256 _amount) external onlyOwner {
-        require(PULL_SUPPLY.sub(pullSpentSupply) >= _amount);
-        require(crowdsaleFinished == false);
-        require(_to != address(0));
-        pullSpentSupply = pullSpentSupply.add(_amount);
-        token.transfer(_to, _amount);
+    function sendAirdrop(address[] _addresses, uint256[] _amounts) external onlyOwner {
+        require(_addresses.length == _amounts.length);
```

Comment:
Do we want to track the addresses we sent the airdrop to, to make sure that we don't double send airdrop to the same address?
Or do we want to leave this logic out of the contract code?

Reply:
Asked it too
We shouldn't